scripts/preference_helper.py

```diff
@@ -0,0 +1,39 @@
+# Adds the default boilerplate code to the preference file.
+
+#!/usr/bin/env python3
```

Comment:
This should be the very first line in the file for it to work, it can't anywhere else.